<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QWidget" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>11</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RNC - Random Name Caller</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f8fa;
font: 10.5pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <widget class="QPushButton" name="btn_startChoosing">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	color: #ffffff;
	background-color: #0064c8;
	border-color: #005db9;
	border-width: 2px;
	border-style: solid;
	border-radius: 8px;
}
QPushButton:disabled 
{
	color: #c8c8c8;
	background-color: #f0f0f0;
	border-color: #f0f0f0;
}
QPushButton:hover
{
	background-color: #106fcd;
	border-color: #0f69be;
}
QPushButton:pressed 
{
	background-color: #217cd2;
	border-color: #1f74c3;
}</string>
   </property>
   <property name="text">
    <string>开始抽取</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text_resultName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>600</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>32</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 32pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>等待抽取</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_modeBox" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	color: #000000;
	background-color: #fafafa;
	border-color: #ebebeb;
	border-style: solid;
}
QPushButton:disabled 
{
	color: #c8c8c8;
	background-color: #f0f0f0;
	border-color: #f0f0f0;
}

QPushButton:hover
{
	background-color:#f0f6fa;
	border-color: #e2e7eb;
}

QPushButton:pressed 
{
	background-color: #ebf0f5;
	border-color: #dde1e6;
}
QPushButton:checked 
{
	color: #ffffff;
	background-color: #0064c8;
	border-color: #005db9;
}</string>
   </property>
   <widget class="QPushButton" name="btn_mode_pseudoRandom">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px 0px 1px 1px;
border-top-left-radius: 8px;
border-bottom-left-radius: 8px;</string>
    </property>
    <property name="text">
     <string>均匀选择</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="modeID" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_completelyRandom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px 0px 1px 0px;</string>
    </property>
    <property name="text">
     <string>完全随机</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="modeID" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_avoidRepeat">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px 1px 1px 0px;
border-top-right-radius: 8px;
border-bottom-right-radius: 8px;</string>
    </property>
    <property name="text">
     <string>避免重复</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="modeID" stdset="0">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="text_progInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>240</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>Made by MucheXD, Version=0.24</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_choosingInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>561</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #c8c8c8;
font: 11.5pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>点击&quot;开始抽取&quot;以启动</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_candidateArea" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>180</y>
     <width>461</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: #c8c8c8;
	background-color: #fafafa;
	border-color: #ebebeb;
	border-style: solid;
	border-width: 2px;
}
QLabel[styleType=&quot;0&quot;]
{
	color: #000000;
	border-color: #0064c8;
}
QLabel[styleType=&quot;1&quot;]
{
	color: #ffffff;
	background-color: #0064c8;
	border-color: #005db9;
}</string>
   </property>
   <widget class="QLabel" name="candidateMember_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>待选成员</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleType" stdset="0">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="candidateMember_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>待选成员</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleType" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="candidateMember_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>待选成员</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleType" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_candidateFlash">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0064c8;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_testTool">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>40</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 200, 200);
border-style: none;
text-decoration:underline;</string>
   </property>
   <property name="text">
    <string>调试</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RNC.qrc"/>
 </resources>
 <connections/>
</ui>
